cypress/support/helpers/visitPage.js

```diff
@@ -17,6 +17,11 @@ export default (path, pageType) => {
     fixture: 'flagpoles/ngas.json',
   });
 
+  cy.setCookie(
+    'FCCDCF',
+    '[["AKsRol_BwEkc3ASfKEKVPeyMAwqICoHzH8NxlVr-Hh0TPFtwU_P6z2pKf8yvnOIlbfkSr5BGpCtal_EyIbeetCsHtGMU9lNGJaJ06kps2FHaY4c62aUnvarsjK7MIWPNR3bVUyNhZyUjTDIlWjH0mn-itSG_3MABGA=="],null,["[[],[],[],[],null,null,true]",1620291054220],["CPFxK7bPFxK7bEsABBENBYCoAP_AAE_AAAwIGWQHgAFAAMAAqABwAEAAKgAZAA0gCIAIsATABPAC-AIEAQgAlABLACkAHsATQAnYBSIC0ALSAXUA4gB-wFkALeAXmAvcBjIDLAMsgIgAVAA4ACAAGkARABFACYAE8AL4AhABLAD2AJ3AWgBaQC6gHEAXmAywAAA","1~1843.202.780.2325.1127.505","B6F8AF01-4E2B-4982-98CF-BC6EA8B00241"],null]',
+  );
```

Comment:
Could we add a comment explaining what this does? In suspect this could be quite a brittle solution as I assume this is Google's cookie format we have no control over?

If this is the best we have though, happy to go with it for now

Reply:
I will add a comment to explain. I don't know any other fix idea at the moment.